tests/lib/rss.test.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * RSS Library Test
+ */
+
+const fs = require('fs')
+const rss = require('../../lib/rss')
+
+describe('lib.rss', () => {
+  let rssResponse;
+
+  beforeAll(async () => {
```

Comment:
This will run once.

Each test should get fresh object to work with.

Reply:
Sure I can do that. Why should we do this, though? The object isn't manipulated at all and the data should be the same.